src/main/java/com/igridin/desktop_calculator/FrameForm.java

```diff
@@ -27,27 +29,27 @@
     static boolean isClassicForm = true;
 
     //Engineering buttons
-    JButton buttonBktOpen = new javax.swing.JButton();
```

Comment:
This is an IDEA-made optimization. It prefers importing a package and then using the short class name, which makes sense.